tests/api_tests/nodes/test_views.py

```diff
@@ -890,21 +891,27 @@ def setUp(self):
         self.basic_auth = (self.user.username, 'password')
         self.project = ProjectFactory(is_public=False, creator=self.user)
         self.pointer_project = ProjectFactory(is_public=False, creator=self.user)
-        self.project.add_pointer(self.pointer_project, auth=Auth(self.user))
-        self.private_url = '/v2/nodes/{}/pointers/'.format(self.project._id)
-        self.private_payload = {'node_id': self.project._id}
+        self.private_url = '{}nodes/{}/pointers/'.format(API_PREFIX, self.project._id)
```

Comment:
I gave you bad advice earlier. Please change API_PREFIX to API_BASE. That is the proper thing to use.